content/docs/command-reference/repro.md

```diff
@@ -51,8 +51,8 @@ There are a few ways to restrict what will be regenerated by this command: by
 specifying specific reproduction [`targets`](#options), or by using certain
 command [options](#options), such as `--single-item` or `--all-pipelines`.
 
-> Note that stages without dependencies are considered _always changed_, so
-> `dvc repro` always executes them.
+> Note that stages without dependencies or outputs are considered _always
+> changed_, so `dvc repro` always executes them.
```

Comment:
Maybe link to https://dvc.org/doc/command-reference/stage/add#--always-changed?

Reply:
link added